apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -208,6 +208,7 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
   const TestComponentView: React.FunctionComponent = () => {
     return (
       <ScrollView
+        keyboardShouldPersistTaps="handled"
```

Comment:
Could you add a small note for why this was required as part of Input changes?

Reply:
Added